src/features/cookies/helpers/useIsCookiesListUpToDate.native.test.ts

```diff
@@ -0,0 +1,121 @@
+import mockdate from 'mockdate'
+
+import { useIsCookiesListUpToDate } from 'features/cookies/helpers/useIsCookiesListUpToDate'
+import * as Firestore from 'libs/firebase/firestore/getCookiesLastUpdate'
+import { storage } from 'libs/storage'
+import { flushAllPromisesWithAct, renderHook } from 'tests/utils'
+
+import Package from '../../../../package.json'
+
+const DAY_IN_MS = 24 * 60 * 60 * 1000
```

Comment:
On a plusieurs endroits ces valeurs de temps (avec `MILLISECONDS_IN_A_DAY`, `DAYS_IN_A_WEEK`, `ONE_YEAR`, etc.), je pense que dans une PR dédiée, on pourrait créer des constantes qu'in utilise partout.

Reply:
Je me note ça